src/toasts/toasts.js

```diff
@@ -0,0 +1,30 @@
+export function showToast(description) {
```

Comment:
Let's change `showToast` to take in an `options` object as an argument. That way we can make it possible for the user to configure things like timeout duration, coloration/styling of the toast, maybe "action button callback", etc.